.gitignore

```diff
@@ -11,6 +11,7 @@
 *.pid.lock
 *.rbc
 *.seed
+.browserslistrc
```

Comment:
This needs to not be gitignored but it does need to be in the npmignore file.

Reply:
Oops.  I think the npm ignore is covered by this: https://github.com/cerner/terra-application/blob/main/package.json#L65-L71 This still needs to be removed from the gitignore though

```suggestion
```

Reply:
Updated here https://github.com/cerner/terra-application/pull/114/commits/438b2e1f36518f86e5b2a7083ae7d8893b692e7e.